.github/workflows/release.yml

```diff
@@ -11,6 +11,43 @@ jobs:
   release:
     name: release ${{ github.event.ref }}
     needs: before-release
-    uses: ansible-community/devtools/.github/workflows/release.yml@main
-    with:
-      pypi: true
+    # unable to use environment with uses/with, basically cannot reuse release pipelines
+    environment: release
+    runs-on: ubuntu-20.04
+
+    env:
+      FORCE_COLOR: 1
+      PY_COLORS: 1
+      TOXENV: packaging
+      TOX_PARALLEL_NO_SPINNER: 1
+
+    steps:
+    - name: Switch to using Python 3.8 by default
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install tox
+      run: >-
+        python3 -m
+        pip install
+        --user
+        tox
+    - name: Check out src from Git
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # needed by setuptools-scm
+    - name: Build dists
+      run: python -m tox
```

Comment:
Where's a separate step for env provisioning?

Reply:
No need. I never used that separation for release. That only makes sense when you have lots of dependencies and/or multiple environments chained, not with two dependencies (wheel, twine).